src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  
+  if (days >= 7) {
+    return cost * days - 50;
+  } else if (days >= 3) {
+    return cost * days - 20;
+  } else {
+    return cost * days;
+  }
```

Comment:
1. move such values as 3, 7, 20 and 50 to variables
2. it's better to avoid using `else if` for better readability. You can use `early return`
Example:
``` 
if(condition1) {
  return result1
}

if(condition2) {
  return result2
}

return resultDefault
```

Reply:
Thanks for review. I making changes!